api.js

```diff
@@ -3,8 +3,18 @@ const service = axios.create({
 });
 
 const api = {
-  getAll: () => null, // Return instead a promise containing all the movies,
-  getOne: id => null, // Return instead a promise containing one movie,
+  getAll: () => {
+    return service.get("movies").then(response => {
+    return response.data;
+    });
+  }, // Return instead a promise containing all the movies,
+
+
+  getOne: id => {

```

Comment:
The argument for an HTTP request should be a URL, so this should be `service.get("movies/${id}")` . When you call this method later, for example with the id 4, it will become `service.get("http://localhost:3000/movies/4")` , because we prefixed the base url when we created the axios instance (see line 2), and then we injected the dynamic id variable into the string url. 
Also possible to do  `service.get("movies/" + id)`